python/lsst/meas/extensions/ngmix/processCoaddsNGMix.py

```diff
@@ -1203,20 +1203,8 @@ class ProcessDeblendedCoaddsNGMixMaxTask(ProcessCoaddsNGMixMaxTask):
     _DefaultName = "processDeblendedCoaddsNGMixMax"
     ConfigClass = ProcessDeblendedCoaddsNGMixMaxConfig
 
-    def run(self, images, ref, replacers, imageId, butler=None, kwargs_butler=None):
-        """
-        make sure we are using the deblended images
-        """
-        check = (
-            replacers is not None
-            and self.cdict['useDeblends'] is True
-        )
-        assert check,\
```

Comment:
Did this check get done elsewhere?

Reply:
I believe it's unnecessary with the construction of the `noiseReplacer`s here: https://github.com/lsst-dm/meas_extensions_ngmix/pull/24/files/1afdddd136b0ae63778e549629851d952750f72a#diff-a1c8a13d1bbe3ba137feaef8a9846493d607e2414e3e39ae6a4f097f1c04f073R148

Reply:
That allows for no replacers, but the check was to make sure the deblending and replacers were used

Reply:
I see. I'll put `assert self.cdict['useDeblends'] is True` into `__init__` then, so that it can fail earlier.